src/libs/protocols/protos/confidential_data_hub.proto

```diff
@@ -28,6 +28,24 @@ message SecureMountResponse {
     string mount_path = 1;
 }
 
+message ImagePullRequest {
+    // - `image_url`: The reference of the image to pull
+    string image_url = 1;
+
+    // - `bundle_path`: The path to store the bundle. After the RPC is
+    // called, Confidential Data Hub will put a `config.json` file
+    // and a mounted `rootfs` directory under the this path. Note
```

Comment:
Who creates the `rootfs` directory?

Reply:
[CDH/image-rs](https://github.com/confidential-containers/guest-components/blob/main/image-rs/src/image.rs#L558) does this.

Reply:
Can we clarify the specification, then? 

> `bundle_path`: [...] This directory is created by the client, and initially empty. The server creates a `rootfs` directory in `bundle_path` and mounts the rootfs into it.

Reply:
nice suggestion. added.